algopy_testing/examples/auction/test_contract.py

```diff
@@ -24,9 +24,8 @@ def test_opt_into_asset(context: AlgopyTestContext) -> None:
     contract.opt_into_asset(asset)
 
     # Assert
-    assert len(context.get_transaction_group()) == 1
     assert contract.asa.id == asset.id
-    inner_txn = context.get_last_submitted_inner_transaction()
+    inner_txn = context.get_last_submitted_itxn_loader().asset_transfer()
```

Comment:
why `_loader`?

Reply:
@robdmoore Regarding this, if I do remove loader wouldn't that be a bit misleading, the intent was to make it obvious that its not gonna return the txn but something that gives you ability to access them.

Reply:
The intellisense guides us on what the properties that are available are though? Loader is a weird concept that we would have to explain, easier and terser to not need to?

Reply:
Or maybe we change it to:

```suggested
    inner_txn = context.last_submitted_itxn().get_asset_transfer()
```

Reply:
Side-note: can the `last_submitted_itxn` bit be a property rather than a method? Or is that weird Python?

i.e.

```suggestion
    inner_txn = context.last_submitted_itxn.get_asset_transfer()
```

Reply:
Also, out of curiosity, what's the syntax when getting an itxn by index? Do we have this same loader abstraction?

i.e. something like

```python
    inner_txn = context.submitted_itxns[2].get_asset_transfer()
```